src/main/dynamo/operations/batchWriteItems.ts

```diff
@@ -29,6 +30,8 @@ export async function batchWriteItems<T extends TaggedModel>(
   deleteItems.forEach((key) => {
     requests.push({
       DeleteRequest: {
+        // @ts-ignore The type is incorrect for `Key`. It expects Record<string, AttributeValue>, but since we're using
+        //  a DynamoDBDocumentClient, it needs to be Record<string, string> instead.
```

Comment:
Writes fail if I pass an object in the format `Record<string, AttributeValue>`. The type in the underlying library is wrong.